client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
```

Comment:
### Nitpick

I'm guessing this particular hex code is already defined in the `variables.scss` why not reference it from there instead.